tests/test_version.py

```diff
@@ -0,0 +1,16 @@
+import re
+import pdbx
+from pdbx import __version__, config
+
+
+def test_version_exists():
+    assert hasattr(pdbx, "__version__")
+
+
+def test_version():
+    assert re.match(r"[0-9]+\.[0-9]+", __version__) or __version__.startswith(
+        "0+untagged"
+    )
+    assert re.match(
+        r"[0-9]+\.[0-9]+", config.VERSION
+    ) or config.VERSION.startswith("0+untagged")
```

Comment:
are `pdbx.__version__` and `pdbx.config.VERSION` always the same? How about `assert __version__ == config.VERSION` ?

Reply:
Added, thanks!